lib/stella/searchable.rb

```diff
@@ -87,7 +88,12 @@ def index_slug
 
       # sets up mappings and settings for index
       def searchable(settings = Stella::Analysis::DEFAULT_SETTINGS, &block)
+        def field(field, opts = {})
+          es_field(field, opts)
+        end
```

Comment:
Ha, yes it does actually! I hadn't realised inner methods weren't locally scoped ([thread](http://stackoverflow.com/questions/4864191/is-it-possible-to-have-methods-inside-methods)) as I generally avoid them on design grounds. I'll update with a spec and think about an alternative implementation.